src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1041,7 +1041,7 @@ void Graph::PushInputData(const std::size_t& index, const ov::SoPtr<ITensor>& in
 
             if (actualDesc->getPrecision() == element::string) {
                 StringMemory ext_mem(getEngine(), ext_tensor_desc, ext_data_ptr);
-                edgeMemory->load(ext_mem);
+                edgeMemory->load(ext_mem, true);
```

Comment:
Should be false I guess

Reply:
Agree, fixed

Reply:
Seems like you haven't pushed the changes yet?

Reply:
Yes, now it should be fine